### PR TITLE
Use HTTPClient that's included in the JDK

### PR DESCRIPTION
**Pull request by slacmshankar**
Remove Apache HTTP components as a dependency.

### CODE REVIEW

File: lib/build.gradle [hunk 1]
```diff
@@ -15,7 +15,7 @@ plugins {
     id("maven-publish")
 }
 
-version = '0.1.1'
+version = '0.0.11'
```
Comment (jacomago) [inline lib/build.gradle]: Why this version number? We should only increment upwards, perhaps 0.2.0 would be better.

File: lib/src/main/java/org/epics/archiverappliance/retrieval/client/RawDataRetrieval.java [hunk 2]
```diff
@@ -71,13 +78,15 @@ public final GenMsgIterator getDataForPVs(
         String getURL = buf.toString();
         logger.info("URL to fetch data is " + getURL);
         try {
-            URL url = new URL(getURL);
-            var data = Request.get(url.toURI()).execute().returnContent();
-
-            BufferedInputStream is = new BufferedInputStream(data.asStream());
-
-            if (is.available() <= 0) return null;
```
Comment (jacomago) [inline lib/src/main/java/org/epics/archiverappliance/retrieval/client/RawDataRetrieval.java:79]: Maybe keep the check that the body isn't empty as well.

**Review by jacomago — CHANGES_REQUESTED** [review]

File: lib/build.gradle (hunk 1 above)
Comment (slacmshankar) [reply lib/build.gradle]: Right you are...

File: lib/src/main/java/org/epics/archiverappliance/retrieval/client/RawDataRetrieval.java (hunk 2 above)
Comment (slacmshankar) [reply lib/src/main/java/org/epics/archiverappliance/retrieval/client/RawDataRetrieval.java:79]: Makes sense. Pushed changes with this back in. Thx for the heads up.

**Review by jacomago — APPROVED** [review]

**Pull request merged**